server/src/test/java/org/candlepin/dto/api/v1/CapabilityTranslatorTest.java

```diff
@@ -34,18 +34,18 @@
     protected CapabilityTranslator capabilityTranslator = new CapabilityTranslator();
 
     @Override
-    protected void initModelTranslator(ModelTranslator modelTranslator) {
+    public void initModelTranslator(ModelTranslator modelTranslator) {
```

Comment:
What's driving the change to increase visibility here? These are intended to be used only by subclasses.

Reply:
It's because we're using the DTOs from the dto.api.v1 package, so it would be justified to be able to use the test utility methods from the same package. Since we're going to have separate DTOs isolated in the dto.rules.v1 package though as you mentioned in the other comment, I'll revert these changes, and either introduce new test classes in that package (rules), or do manual setup/validation if they're minimal enough.

Reply:
ack